pkg/pillar/devicenetwork/dns.go

```diff
@@ -131,6 +131,10 @@ func ResolveWithPortsLambda(domain string,
 				copy(srcIPCopy, srcIP)
 				countDNSRequests++
```

Comment:
len(wg) == countDNSRequests, so technically we don't need it.

Reply:
`wg` is not an array or a string, so I am not sure what `len(wg)` should do